pkg/controller/job/controller.go

```diff
@@ -353,13 +353,13 @@ func newCondition() experimental.JobCondition {
 	}
 }
 
-func getStatus(pods []api.Pod) (successful, unsuccessful int) {
+func getStatus(pods []api.Pod) (successful, failed int) {
 	successful = filterPods(pods, api.PodSucceeded)
-	unsuccessful = filterPods(pods, api.PodFailed)
+	failed = filterPods(pods, api.PodFailed)
 	return
 }
 
-func (jm *JobController) manageJob(activePods []*api.Pod, successful, unsuccessful int, job *experimental.Job) int {
+func (jm *JobController) manageJob(activePods []*api.Pod, successful, failed int, job *experimental.Job) int {
```

Comment:
I guess "failed" is unused in this function, but probably fine to keep it (I'm surprised Go doesn't complain?)